packages/material-ui/src/Alert/alertClasses.js

```diff
@@ -0,0 +1,23 @@
+import { generateUtilityClass, generateUtilityClasses } from '@material-ui/unstyled';
+
+export function getAlertUtilityClass(slot) {
+  return generateUtilityClass('MuiAlert', slot);
+}
+
+const alertClasses = generateUtilityClasses('MuiAlert', [
+  'root',
```

Comment:
We need to add the
```suggestion
  'root',
  'action',  
  'icon',
  'message',
```

We need to add the slots here too